rust/crd/src/lib.rs

```diff
@@ -19,7 +19,7 @@ use stackable_operator::{
     config::{fragment, fragment::Fragment, fragment::ValidationError, merge::Merge},
     k8s_openapi::apimachinery::pkg::api::resource::Quantity,
     kube::{runtime::reflector::ObjectRef, CustomResource},
-    product_config_utils::{ConfigError, Configuration},
+    product_config_utils::{Configuration, Error as ConfigError},
```

Comment:
nit: I'm not a big fan of renamed imports. I would like to see the use of fully qualified import paths or something like:

```rust
use stackable_operator::product_config_util::{self, Configuration};

let error = product_config_util::Error {};
```

Reply:
https://github.com/stackabletech/hive-operator/pull/462/commits/2c80e5ba6b5f932bd7d97013bac33bc68cde76c2